### DISCUSSION TITLE
Is there any equivalent to AfterLoad() in TypeOrm?

### DISCUSSION THREAD

**Discussion by MuhammedKpln** [prisma/prisma#10444] [Q&A] [answered]
Hello, is there any equivalent on TypeORM's entity listeners?

While you can use middleware to mock data before sending it do data, it is not possible to mock data when you try to get via find() while you can do this in typeorm like [this with AfterLoad() decorator](https://github.com/typeorm/typeorm/blob/master/docs/listeners-and-subscribers.md#afterload)

**Comment by georgekrax** [comment, marked as answer]
Here is an [answer](https://github.com/prisma/prisma/discussions/10595#discussioncomment-1769234) to the same question of mine. You can always use `middlewares`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
